Sources/SwiftGtk/enums/NotebookTab.swift

```diff
@@ -0,0 +1,29 @@
+import CGtk
+
+/// https://docs.gtk.org/gtk3/enum.NotebookTab.html
+public enum NotebookTab {
+    case first
```

Comment:
Add documentation for these cases if possible

Reply:
These have no descriptions in the documentation.